storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -921,6 +923,27 @@ public static void setQueueKeepSortedOrder(SortOrder sortOrder) {
                 .apply();
     }
 
+    /**
+     * Returns the sort order for the downloads.
+     */
+    public static SortOrder getDownloadsSortedOrder() {
```

Comment:
The UserPreferences class is already quite long (and almost the entire app has a dependency on it). I would instead move this to the downloads fragment and store it in its own preferences. Similar to how we do it for the "columns" setting on the subscriptions screen.